how-to/author-guide.md

```diff
@@ -268,23 +271,34 @@ are usually not accepted by JOSS. Be sure to review JOSS's
 before writing up a paper about your package.
 ```
 
-## Post review - now what?
+## Post review - welcome to the pyOpenSci community!
+
+Congratulations! Once your package has been accepted into the pyOpenSci ecosystem, you'll be invited to join our [community Slack](https://join.slack.com/t/pyopensci/shared_invite/zt-39qitgkqb-gZTIo79xCJhS5kSxW1yNfg) where you can connect with other package maintainers, get help with maintenance questions, and stay updated on community developments.
+
+### Ongoing support from pyOpenSci
+
+We're committed to supporting you throughout your package's maintenance journey:
 
-Once you have been accepted into the pyOpenSci ecosystem, you will
-be invited to join our Slack channel.
+**Annual check-ins**: We'll reach out each year to see how your package is doing and learn about any updates we can help highlight through our blog, social media, or newsletter.
 
-We also will keep in touch with you, periodically checking in to ensure that package maintenance is going well and to better understand ways in which pyOpenSci can support you.
+**Community resources**: Take advantage of our Slack community to connect with other maintainers, share experiences, and get advice on common maintenance challenges. We're also building additional resources and tools to help package maintainers succeed.
 
-If at any time, you need to step down from maintaining your package,
-or you need help with maintenance, please let us know - preferably
-in advance. We will try to help you by either:
+**Maintenance assistance**: If you have specific maintenance challenges, our Slack community is a great place to ask questions. There is a lot of packaging and community expertise in our vibrant community!
 
-- Finding a new maintainer to take over your project (or additional maintainers to support maintenance) or
-- Sunsetting your package.
+### When you need help or want to step down
```

Comment:
I've reworded this and linked to our archive policy page so the policy isn't in the guidebook twice. The policy page is where we had a formal review and discussion with the astropy community and the pyOpenSci community so that language is vetted. Whereas this language was slightly different and inconsistent.